Cargo.toml

```diff
@@ -9,3 +9,4 @@ exe = "0.5.6"
 num = "0.4.3"
 num-derive = "0.4.2"
 num-traits = "0.2.19"
+rayon = "1.10.0"
```

Comment:
What is this crate used for?

Reply:
This made the multithreading for parsing the PE files (see the funciton par_iter) not make me want to through my computer through a wall and fight with the borrow checker